src/gnssro/gnssro_bufr2ioda_generic.f90

```diff
@@ -0,0 +1,496 @@
+   call check(nf90_put_att(grpid_metadata, varid_epochtime, "units", "seconds since 1970-01-01T00:00:00Z"))
+
+   call check(nf90_def_var(grpid_metadata, "timeOffset", NF90_INT, nlocs_dimid, varid_tinc))
+   call check(nf90_def_var_fill(grpid_metadata, varid_tinc, 0, i_missing))
+   call check(nf90_def_var_deflate(grpid_metadata, varid_tinc,      &
+                                    & shuffle=1, deflate=1, deflate_level=deflate_level))
+   call check(nf90_put_att(grpid_metadata, varid_tinc, "longname", "time increment from the start time of the occultation"))
+   call check(nf90_put_att(grpid_metadata, varid_tinc, "units", "second"))
+
+   call check(nf90_def_var(grpid_metadata, "sequenceNumber", NF90_INT, nlocs_dimid, varid_recn))
+   call check(nf90_def_var_fill(grpid_metadata, varid_recn, 0, i_missing))
+   call check(nf90_def_var_deflate(grpid_metadata, varid_recn,      &
+                                    & shuffle=1, deflate=1, deflate_level=deflate_level))
+   call check(nf90_put_att(grpid_metadata, varid_recn, "longname", "GNSS RO profile identifier"))
+   call check(nf90_put_att(grpid_metadata, varid_recn, "units", "1"))
+
+   call check(nf90_def_var(grpid_metadata, "satelliteConstellationRO", NF90_INT, nlocs_dimid, varid_sclf))
+   call check(nf90_def_var_fill(grpid_metadata, varid_sclf, 0, i_missing))
+
+   call check(nf90_def_var_deflate(grpid_metadata, varid_sclf,       &
+                                    & shuffle=1, deflate=1, deflate_level=deflate_level))
+   call check(nf90_put_att(grpid_metadata, varid_sclf, "longname", &
+      & "GNSS satellite classification, e.g., 401=GPS, 402=GLONASS"))
+   call check(nf90_put_att(grpid_metadata, varid_sclf, "units", "1"))
+
+   call check(nf90_def_var(grpid_metadata, "satelliteTransmitterId", NF90_INT, nlocs_dimid, varid_ptid))
+   call check(nf90_def_var_fill(grpid_metadata, varid_ptid, 0, i_missing))
+   call check(nf90_def_var_deflate(grpid_metadata, varid_ptid,       &
+                                    & shuffle=1, deflate=1, deflate_level=deflate_level))
+   call check(nf90_put_att(grpid_metadata, varid_ptid, "longname", "GNSS satellite transmitter identifier (1-32)"))
+   call check(nf90_put_att(grpid_metadata, varid_ptid, "units", "1"))
+
+   call check(nf90_def_var(grpid_metadata, "satelliteIdentifier", NF90_INT, nlocs_dimid, varid_said))
+   call check(nf90_def_var_fill(grpid_metadata, varid_said, 0, i_missing))
+   call check(nf90_def_var_deflate(grpid_metadata, varid_said,       &
+                                    & shuffle=1, deflate=1, deflate_level=deflate_level))
+   call check(nf90_put_att(grpid_metadata, varid_said, "longname", &
+      & "Low Earth Orbit satellite identifier, e.g., COSMIC2=750-755"))
+   call check(nf90_put_att(grpid_metadata, varid_said, "units", "1"))
+
+   call check(nf90_def_var(grpid_metadata, "occulting_sat_is", NF90_INT, nlocs_dimid, varid_siid))
```

Comment:
I think this should be named `instrumentIdentifier`.

Reply:
Thanks for the catch. 
Making notes here: this also need to be reflected in "ioda-upgrade-v2-to-v3"
by ioda/share//ioda/yaml/validation/ObsSpace.yaml